Week3/homework/step2-3.js

```diff
@@ -30,9 +32,13 @@ console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 function repeatStringNumTimesWithDoWhile(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
+  if (num <= 0) return result;
+  let z = 0;
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  do {
+    z++;
+    result += str;
+  } while (z < num);
```

Comment:
👍 for logic but I wonder why did you not have the initial `if` condition inside `for` and `while` loop implementations? 🤔